docker-compose.yml

```diff
@@ -21,6 +21,7 @@ x-gunicorn-master: &x-gunicorn-master
     - SQL_HOST=database
     - SQL_USER=postgres
     - SQL_DATABASE=postgres
+    - EMAIL_PROVIDER=DISABLED
```

Comment:
Does this need to be specified?

Reply:
No, the default value is `DISABLED`. I left it in to make it more obvious to change, but removing it wont break anything.

Reply:
Perhaps set it to `CONSOLE` so we get local logs when emails are sent?

Reply:
Fair, will do that